R/deployments.R

```diff
@@ -173,11 +174,12 @@ deploymentFile <- function(appPath, name, account, server) {
   file.path(accountDir, paste0(name, ".dcf"))
 }
 
-deploymentRecord <- function(name, account, server, appId, bundleId, url, when,
-                             metadata = list()) {
+deploymentRecord <- function(name, title, account, server, appId, bundleId, url,
+                             when, metadata = list()) {
   # compose the standard set of fields and append any requested
   as.data.frame(c(
       list(name = name,
+           title = if (is.null(title)) "" else title,
```

Comment:
Does this have the side-effect of producing empty strings instead of null values for API requests on subsequent deploys? I would prefer to see null over an empty string.